pkg/api/handlers/bokers_handler.go

```diff
@@ -0,0 +1,36 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/inspr/inspr/pkg/api/models"
+	"github.com/inspr/inspr/pkg/rest"
+	"go.uber.org/zap"
+)
+
+// BrokerHandler - contains handlers that uses the BrokerManager interface methods
+type BrokerHandler struct {
+	*Handler
+}
+
+// NewBrokerHandler - returns the handle functions that regard brokers
+func (handler *Handler) NewBrokerHandler() *BrokerHandler {
+	return &BrokerHandler{
+		handler,
+	}
+}
+
+// HandleGet returns the get handler for brokers
+func (bh *BrokerHandler) HandleGet() rest.Handler {
+	logger.Info("handling Brokers get request")
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		brokers := &models.BrokersDi{
+			Installed: bh.Brokers.GetAll(),
+			Default:   string(bh.Brokers.GetDefault()),
+		}
```

Comment:
I just noticed a thing that bothered me. The `get` method defined in `cmd/insprd/memory/brokers/brokers.go` never returns an error, instead it panics if no Broker structure is found.
I really think that method must be changed and, by doing so, also change this one here. The only time we use the panic function in our project is when necessary env vars are not found, so using it in a `insprd/memory` function doesn't seem right.

Reply:
the get method doesnt acctually need that panic, it is mostly a formality but it will never execute